utility/src/main/java/org/oppia/android/util/parser/BitmapBlurrer.kt

```diff
@@ -6,52 +6,48 @@ import android.renderscript.Allocation
 import android.renderscript.Element
 import android.renderscript.RenderScript
 import android.renderscript.ScriptIntrinsicBlur
-import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool
-import com.bumptech.glide.load.resource.bitmap.BitmapTransformation
-import java.security.MessageDigest
-
-// The radius of the blur, a float between 0 and 25. It defines the value of the standard
-// deviation to the Gaussian function.
-private const val BLUR_RADIUS = 20f
 
 /**
- * [BlurTransformation] is a bitmap transformation that blurs an image using RenderScript.
- *
- * The following [tutorial](https://futurestud.io/tutorials/glide-custom-transformation)
- * was used as a reference, as well as this [article](https://stackoverflow.com/a/23119957).
+ * The radius of the blur, a float between 0 and 25. It defines the value of the standard deviation
+ * to the Gaussian function.
  */
-class BlurTransformation(private val context: Context) : BitmapTransformation() {
+private const val BLUR_RADIUS = 20f
 
+/** Utility used to blur [Bitmap]s. */
+internal class BitmapBlurrer(private val context: Context) {
```

Comment:
Though this class name makes sense to me but reading it sounds a bit awkward.

Reply:
Might be because 'blurrer' isn't a word. :) Any suggestions for alternative names?

Reply:
'Impl' doesn't really fit since it's not implementing an interface/API. 'BlurEffect' is a bit off, too, since 'effect' actually means something in Android and this isn't that (https://developer.android.com/reference/android/media/effect/Effect).